### DISCUSSION TITLE
Roadmap / status

### DISCUSSION THREAD

**Discussion by cbruyndoncx** [danielmiessler/Fabric#77] [General]
I just discovered this opensource fabric ai framework, recently learned about m$ fabric but that is a totally different beast.

Anyhow, i was browsing the repo and Rreadme and came across some not functional links and things mentioned that dont seem to exist..
I also saw some directory restructuring going on, and a lot of activity.

So question is, is it usefull for me to help with these corrections ?

**Comment by danielmiessler** [comment]
Give it a few days, we're doing some restructuring. Thank you!
